src/ui/mainwindow.cpp

```diff
@@ -2030,48 +2042,6 @@ void MainWindow::on_actionPrint_Now_triggered()
     QPrinter printer(QPrinter::HighResolution);
     currentEditor()->print(&printer);
 }
-/*
-void MainWindow::on_actionLaunch_in_Chrome_triggered()
-{
-    QUrl fileName = currentEditor()->fileName();
-    if (!fileName.isEmpty()) {
-        QStringList args;
-        args << fileName.toString(QUrl::None);
-        QProcess::startDetached("google-chrome", args);
-    }
-}
-*/
-QStringList MainWindow::currentWordOrSelections()
```

Comment:
I could reimplement it and keep it for now, yea... shouldn't be an issue.... I haven't the slightest clue if it works, however.